communicator/room.go

```diff
@@ -253,7 +250,39 @@ func (r *Room) startStorePhase() {
 			r.Shutdown("Failed fetching units")
 			return
 		}
-		state.Store = units
+
+		state.Store = nil
+		for _, u := range units {
```

Comment:
Nit: You could abstract out the GetStoreUnit and GetQuestion to GetStoreUnitWithQuestion, iterate over QuestionDifficulty and build the map dynamically. Would shorten the code considerably and react to adding or removing QuestionDifficulties